new-structure/aspnet-core/build/build-with-ng.sh

```
@@ -0,0 +1,34 @@
+echo " Done. -- Building docker image $ABP_HOST..."
+echo ""
+echo ""
+
+# echo " Pushing docker image $ABP_HOST..."
+# docker push $ABP_HOST
+# echo " Done. -- Pushing docker image $ABP_HOST..."
+# echo ""
+# echo ""
+
+cd ..
+cd angular/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)